samples/drivers/led_lpd8806/boards/96b_carbon.conf

```diff
@@ -2,4 +2,3 @@ CONFIG_SPI_STM32=y
 CONFIG_SPI_STM32_INTERRUPT=y
 CONFIG_SPI_2=y
 CONFIG_SPI_2_NAME="lpd8806_spi"
-CONFIG_SPI_2_IRQ_PRI=2
```

Comment:
The commit log does not explain why these should be removed; can you explain why?

Presumably if an application sets a value specifically, it may be a necessary override of the default. Did you check to make sure you're just removing default settings?

Reply:
If we use DT to get the *_IRQ_PRI, we can't override it using Kconfig. We need to use a BOARD_NAME.overlay file.

> Presumably if an application sets a value specifically, it may be a necessary override of the default. Did you check to make sure you're just removing default settings?

You are right. I didn't check if this value is just a default value or it is necessary for the application. I don't have the hardware to check it, so I will try to ask codeowner for review.

Reply:
>  > Presumably if an application sets a value specifically, it may be a necessary override of the default. Did you check to make sure you're just removing default settings?

> You are right. I didn't check if this value is just a default value or it is necessary for the application. I don't have the hardware to check it, so I will try to ask codeowner for review.

And the samples codeowner is you. I apologize for not checking it with you first. If the value is necessary, I will update the PR with the needed overlay files.

Reply:
I went back and looked at the wire protocols for these drivers.

I think this lpd8806 sample is OK to use the SoC-provided IRQ PRI value (5), but I'm less sure what to do with the ws2812 sample.

(The ws2812 driver plays tricks with SPI to implement a one-wire real-time protocol, so the sample will probably work, but giving higher priority to the SPI IRQ and driver thread is important for applications with multiple tasks.)

This is a personal time driver for me so I will test tonight after work, but my guess is the default will be OK, but I'll need to add extra documentation for this driver about adjusting the IRQ priority using device tree overlays after your series goes in.

Reply:
@ydamigos I didn't make time to test, but I am OK with moving these to the default SoC provided values since these drivers don't have many users right now. I think I need to send those documentation patches about configuring the IRQ priority no matter what. If I run into problems, I'll add overlays to the samples myself. Thanks for the pointer on how to fix any issues.